packages/ndla-editor/src/structure/ItemNameBar.js

```diff
@@ -68,6 +69,23 @@ const ItemTitleButton = styled.button`
   }
 `;
 
+const RoundIcon = ({ smallIcon, ...rest }) => (
+  <StyledIcon {...rest}> {smallIcon}</StyledIcon>
+);
+
+const StyledIcon = styled.button`
+  display: flex;
+  align-items: center;
+
+  border: 0;
+  background: 0;
```

Comment:
Er dette vanlig css praksis? har aldri sett det gjort slik før

Reply:
Evt. hva gjør den? For å gjøre bakgrunnen gjennomsiktig ser jeg at de har brukt 'background-color: transparent;' andre steder i frontend-packages

Reply:
Var iallefall brukt i ItemTitleButton, har ingen bruk for en bakgrunn på det ikonet.